Riot/Modules/Room/RoomViewController.m

```diff
@@ -500,16 +500,6 @@ - (void)viewWillAppear:(BOOL)animated
         
         [self setBubbleTableViewContentOffset:CGPointMake(-self.bubblesTableView.mxk_adjustedContentInset.left, -self.bubblesTableView.mxk_adjustedContentInset.top) animated:YES];
     }];
-    
-    if ([self.roomDataSource.roomId isEqualToString:[LegacyAppDelegate theDelegate].lastNavigatedRoomIdFromPush])
-    {
-        [self startActivityIndicator];
-        [self.roomDataSource reload];
-        [LegacyAppDelegate theDelegate].lastNavigatedRoomIdFromPush = nil;
-        
-        notificationTaskProfile = [MXSDKOptions.sharedInstance.profiler startMeasuringTaskWithName:AnalyticsNoficationsTimeToDisplayContent
```

Comment:
When we remove this we'll lose `notificationTaskProfile`. If this is the intention, we may/should also clean `notificationTaskProfile` and `AnalyticsNoficationsTimeToDisplayContent` related code pieces.

Reply:
Not sure what to do ... What do you think about it @manuroe ?

Reply:
@ismailgulek is right. Either we clean everything or we keep it.

I did not think about the analytics behind. Let's keep everything for the moment because we are still doing things around the NSE.
Maybe, analytics graphs will be able to help us to detect regressions on it.